### DISCUSSION TITLE
Open Discussion

### DISCUSSION THREAD

**Discussion by Asy93** [Asy93/AWE-Grunker-s-Finest#2] [General]
Talk here

**Comment by Boowanda** [comment]
The thing I've noticed with the meta grids - the small, elite, uber agile, high shield grids that everyone loves - Is that there is a kinetic option for them (rails), but not really a practical energy option. Once they pass a certain g/s acceleration, they become effectively unhittable by anything but a hitscan gun or railgun. Conventional weapons are just too slow, missiles are too slow, the laser options we have get outranged every time, the fixed option (prometheus) is disproportionately much more difficult to use (considering all the factors that go into making a shot) and requires scripts most people aren't familiar with to be remotely viable as a counter. 

I realize the meta evolves as a result of balancing, but the current gap in what it takes to kill those grids, compared to what it takes to for those grids to win, seems ginormous. I just think making missiles faster (increasing whatever variable would allow them to be more effective at tracking and hitting dodge grids) or having a hit scan energy weapon for the 2.5km-5km range would at least help in closing that gap and levelling the playing field. I'm more than happy to elaborate on specifics if the issue gains any traction.

**Comment by Starfleck** [comment]
Warrior --------- Increase rate of fire from 420 -> 700.
Artemis --------- Decrease reload from 600 -> 300.
Spartan --------- Increase muzzle velocity from 1400 -> 2200. Increase Comp Damage from 1x -> 2x. Increase Heavy Armor damage from 1x -> 3x.
Nucleon --------- Decrease projectiles by half. Increase Damage from 200 -> 3600.
Shotgun --------- Decrease Shield damage from 5x -> 2.5x.
Strike Cannons -- Muzzle Velocity xxx -> 1200 (Don't forget whirlwind as well)
Railguns -------- Falloff MinMult 0.1 -> 0.3
Shock Cannons --- Decrease Min Damage Range from ~3km -> 1.5km. (Halve MinArmingTime on all)
Helios ---------- Decrease Min Damage Range from ~3km -> 1.5km.
Thrasher -------- Increase Inventory size by 3x.
Punisher -------- Decrease Inventory size by 0.5x (Halve)
Vulcan ---------- Increase base damage from 325 -> 400.

^^^ Targets for midrange balance update, April '23. Contributors to the discussion _itbemeagain, OldKing, TumorSVT, Starfleck, TrueTurnIP, Blatantguitar55_. These changes are mostly to bring lackluster weapons (hopefully) on par with the ones currently being used to good effect, bringing 1.5-3km back into the picture versus the brawler meta.

Next up to look at are: Chord, Flak (aoe damage), Helios (speed), Medusa, Aurora, Sunderer, Echo, Astraeus, Reaper, and Flare.
We'll cover these one or two at a time asap.
